### PR TITLE
bug 1710427 specify number of CRS

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1710427

### CODE REVIEW

File: modules/installation-approve-csrs.adoc [hunk 1]
```diff
@@ -19,20 +19,34 @@ signing requests (CSRs) for the machines that you added.
...
+<2> A server request CSR.
++
+In this example, two machines are joining the cluster.
+
+. After all of the pending CSRs for the machines you added are in `Pending`
+status, approve the CSRs for your cluster machines:
++
+[NOTE]
+====
+Because the CSRs rotate automatically, approve your CSRs within half an hour
```
Comment (abhinavdahiya) [inline modules/installation-approve-csrs.adoc]: cc @sjenning @rphillips

Comment (abhinavdahiya) [reply modules/installation-approve-csrs.adoc]: When trying to get a certificate (client/server), kubelet (node) creates a CSR and waits for it to be Approved and Issued. If the CSR is not approved for _some time_ (Pod team can provide more guidance on what this period is), kubelet (node) creates a new request for certificate with new CSR.

So there's a chance that there might be more CSR objects in Pending for the same purpose.

I'm hoping Pod Team (cc @sjenning ) can provide information on the exact behavior.

Comment (rphillips) [reply modules/installation-approve-csrs.adoc]: the _some time_ the kubelet will wait for a certificate to be issues is 1 hr.

Comment (kalexand-rh) [reply modules/installation-approve-csrs.adoc]: Thanks! I've updated the note to say that it takes an hour for the certs to rotate and that if you wait longer than that, you might have to approve more certs.

**Comment by kalexand-rh** [comment]
This change was approved on the source bug. Merging.

**Pull request merged**